config/test.exs

```diff
@@ -17,3 +17,6 @@ config :elixir_bench, ElixirBench.Repo,
   database: "elixir_bench_test",
   hostname: "localhost",
   pool: Ecto.Adapters.SQL.Sandbox
+
+# Github client for getting repository configuration file bench/config.yml
```

Comment:
Not sure we need this comment in every file, I feel like this might be better as module documentation